db/migrate/20220913100826_add_project_code_index.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddProjectCodeIndex < ActiveRecord::Migration[6.1]
+  def up
+    ActiveRecord::Base.connection.execute(
+      "CREATE INDEX index_projects_on_project_code ON "\
+      "projects using gin (('PR'::text || id) gin_trgm_ops);"
+    )
+  end
+
+  def down
+    remove_index :projects, name: 'index_projects_on_project_code'
+  end
+end
```

Comment:
This was done in a another PR, so just remove this migration.

Reply:
Probably best to rollback first with rails db:migrate:down 20220913100826, then delete